pronto-golang.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files      = ['LICENSE', 'README.md']
   s.require_paths         = ['lib']
 
-  s.add_runtime_dependency('pronto', '~> 0.9.0')
+  s.add_runtime_dependency('pronto', '> 0.9.0', '< 0.11.0')
```

Comment:
`> 0.9.0` disallows `0.9.0`, while `~> 0.9.0` means `>= 0.9.0, < 0.10`

Please adapt it to `>= 0.9.0`

Reply:
done